src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,491 @@
+package strawman
+package collection.mutable
+
+import scala._
+import scala.collection.{generic, mutable}
```

Comment:
In that case I will wait till internal API freeze...

Also do you know why the build is failing? It seems dotty does not support default arguments in the constructor??